### PR TITLE
add a option g:asyncomplete_auto_completeopt

### PR DESCRIPTION
**Pull request by bbchung**
To avoid duplicated add completeopt.

### CODE REVIEW

File: autoload/lsc/complete.vim [hunk 1]
```diff
@@ -125,8 +125,9 @@ function! s:SuggestCompletions(completion) abort
     let base = getline('.')[start - 1:col('.') - 2]
     let suggestions = s:FindSuggestions(base, a:completion)
   endif
-  setl completeopt-=longest
-  setl completeopt+=menu,menuone,noinsert,noselect
+  if (g:lsc_auto_completeopt == 1)
+    setl completeopt=menuone,noinsert,noselect
```
Comment (natebosch) [inline autoload/lsc/complete.vim]: The `+=` was intentional so that the `preview` option could remain untouched.

Comment (bbchung) [reply autoload/lsc/complete.vim]: It seems the completeopt keep growing under some condition. For example if I use vim-lsc and asyncomplete.vim (It's also modify the completeopt, https://github.com/prabirshrestha/asyncomplete.vim ) together.

**Comment by natebosch** [comment]
@bbchung - what do you think about https://github.com/natebosch/vim-lsc/pull/96 ?

**Comment by bbchung** [comment]
@natebosch I like #96, thanks!

**Pull request closed without merging**